test/cli-test.js

```diff
@@ -7,13 +7,31 @@ var cliTxt = path.resolve(_files, 'cli.txt');
 var cliPartialsTxt = path.resolve(_files, 'cli_with_partials.txt');
 var moduleVersion = require('../package').version;
 
-var exec = require('child_process').exec;
+var child_process = require('child_process');
+
+function changeForOS(command) {
+
+  if(process.platform === 'win32') {
+    return command.
+      replace(/bin\/mustache/g, 'node bin\\mustache').
+      replace(/\bcat\b/g, 'type').
+      replace(/\//g, '\\');
```

Comment:
Declaring a new var into which you make the modifications could solve both points:
```js
if (...) {
  var winCommand = command
    .replace(...)
    .replace(...)
    .replace(...)
  return winCommand
}
```
I've been writing JS without semicolons for over a year and I didn't have any problem, I swear! :)